File: 3830-find-closest-person/find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        xSteps = abs(z-x)
        ySteps = abs(z-y)
        if xSteps == ySteps:
            return 0
        return 1 if xSteps < ySteps else 2

File: 3830-find-closest-person/find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        int xSteps = Math.abs(x-z);
        int ySteps = Math.abs(y-z);
        if (xSteps == ySteps){
            return 0;
        } else if (xSteps < ySteps){
            return 1;
        } else{
            return 2;
        }
    }
}